deployments/infrastructure-target-aws.nix

```diff
@@ -4,42 +4,34 @@ with (import ./../lib.nix);
 let org = "IOHK";
     region = "eu-central-1";
     accessKeyId = IOHKaccessKeyId;
-in rec {
-  hydra = { config, pkgs, resources, ... }: {
-    imports = [
-      ./../modules/amazon-base.nix
-    ];
-
-    deployment.ec2 = {
-      inherit accessKeyId;
-      instanceType = mkForce "r3.2xlarge";
-      ebsInitialRootDiskSize = mkForce 200;
-      associatePublicIpAddress = true;
-      securityGroups = [
-        resources.ec2SecurityGroups."allow-deployer-ssh-${region}-${org}"
-        resources.ec2SecurityGroups."allow-public-www-${region}-${org}"
-      ];
-    };
-  };
-
-  hydra-build-slave-1 = hydra;
-  hydra-build-slave-2 = hydra;
+    generic-infra-host =
+      { ebsInitialRootDiskSize
```

Comment:
No function wrapping of NixOS modules anymore please :)

Reply:
Oh, right : -)

By the way, can you suggest a simple way to achieve the same goal?

Reply:
Add nixos configuraition as it was before, if you have a common setting, use `imports`

Reply:
Drat, that's what I was afraid to hear : -)

To be honest, personally this is hard to see as an improvement (splits related things away), so I'll just inline things back.

Reply:
Done!